src/model/scripts/lighting_training/train.py

```diff
@@ -61,7 +61,13 @@
         "--checkpoint",
         type=bool,
         default=False,
-        help="gradient checkpoint only available for GPTJ",
+        help="gradient checkpointing",
+    )
+    parser.add_argument(
+        "--check_position",
+        type=bool,
+        default=False,
+        help="False (model) | True (self-attentions only)",
```

Comment:
I think this line is too weird. Is it better if we use `--checkpoint_only_attention`?

Reply:
done